config/config.msft.yaml

```diff
@@ -166,6 +166,8 @@ clouds:
         imageTag: ecd15ad
         imageRepo: app-sre/uhc-clusters-service
         azureOperatorsManagedIdentities:
+          controlPlaneOperator:
+            roleName: Contributor
```

Comment:
For dev at least we should have a more narrow set of permissions as with the other operators. For now this is ok but we should fix it. The work is tracked in https://issues.redhat.com/browse/ARO-14868.
The same applies to the capz operator.

Reply:
Noted, when the work is complete we need to update the custom roles in dev and then we can remove contributor.